### DISCUSSION TITLE
Welcome to spring-statemachine Discussions!

### DISCUSSION THREAD

**Discussion by jvalkeal** [spring-attic/spring-statemachine#908] [General]
## Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
  * Use discussions instead of posting questions into issues.

**Comment by mehmetsalgar** [comment]


**Reply by jvalkeal** [reply]
Would you mind posting this as a new topic/post as this was kinda welcome post.

**Reply by mehmetsalgar** [reply]
Sure

**Comment by nyemiks** [comment]
Very welcome addition.

**Comment by ShvetsovDV** [comment]
@jvalkeal 
Take a look PR , please!
https://github.com/spring-projects/spring-statemachine/pull/998
